client/my-sites/plans-features-main/index.tsx

```diff
@@ -838,6 +840,7 @@ const PlansFeaturesMain = ( {
 										coupon={ coupon }
 										currentSitePlanSlug={ sitePlanSlug }
 										generatedWPComSubdomain={ resolvedSubdomainName }
+										hideSpotlightPlan={ hideSpotlightPlan }
```

Comment:
Suggestion: instead of introducing another prop, you can also pass `gridPlanForSpotlight` as `undefined`.

Reply:
I had tried that, but it will show the spotlight plan (with "Your Plan" badge) in the grid which I wanted to avoid :(

<img width="1188" alt="image" src="https://github.com/user-attachments/assets/302890fb-ddb7-45f5-a920-e045010946a6" />